data/json/items/fake.json

```diff
@@ -102,5 +102,14 @@
         "copy-from": "fake_item",
         "name": "boulder anvil",
         "qualities": [ [ "ANVIL", 1 ] ]
-    }
+    }, 
+    {
+        "id": "fake_smoke_plume",
+        "copy-from": "fake_item",
+        "type": "GENERIC",
+        "name": "smoldering embers",
+        "description": "A handful of smoldering embers emitting smoke.  They are fading away even when you look at them.",
```

Comment:
I am not sure if the player can see this item when playing. Perhaps add some line like "this is a bug if you see it" to inform the player?

Reply:
If a player destroyes the active rack there is a sliver of time before process() kills the embers where it can be seen. In this regard I have not decided to do that, rather I want the player to see current description that suggests that they wil disappear instantly.